cmd/ocm-support/registryCredentials/cmd.go

```diff
@@ -0,0 +1,21 @@
+package registrycredentials
+
+import (
+	"ocm-support-cli/cmd/ocm-support/registryCredentials/delete"
+	"ocm-support-cli/cmd/ocm-support/registryCredentials/show"
+
+	"github.com/spf13/cobra"
+)
```

Comment:
consider changing the order of imports (third party packages should come before the local ones)